django_dictiterators/apps.py

```diff
@@ -2,4 +2,4 @@
 
 
 class DjangoTemplateIteratorsConfig(AppConfig):
```

Comment:
Would it make sense to change the class name too? The old name doesn't make sense anymore.
```suggestion
class DjangoDictIteratorsConfig(AppConfig):
```